.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+# (Prefix-TicketNumber) : Title of the pull request with ticket number
+
+## Description
+Please write here the description of the pull request.
+
+## Type of change
+
+Please check only the relevant items of this ticket.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+- [ ] Documentation (adds/edits current documentation)
+- [ ] Build (changes the build process)
+
+# How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+# Checklist:
```

Comment:
Is this really necessary? Much of these points are automated, and the output of our various checks are visible in GitHub Actions.

Reply:
oh, in that case let me lower the content, since the template I designed was thought to be detailed as possible.